snap/snapcraft.yaml

```diff
@@ -229,7 +229,8 @@ parts:
     override-build: *arch-restrict
     override-stage: *arch-restrict
     override-prime: *arch-restrict
-    build-snaps: *go
+    build-snaps:
+      - go/1.23/stable
```

Comment:
What is the difference? 

https://github.com/canonical/docker-snap/blob/ac53baeb525611ab9d70178ed1b3e7a5229e8d1f/snap/snapcraft.yaml#L165